File: java/parser/src/main/java/gov/lbl/parser/domain/Term.java
package gov.lbl.parser.domain;

import java.util.Collection;

public class Term {
    private Collection<Factor> factors;
    private Collection<String> mul_ops;

    public Term(Collection<Factor> factors, Collection<String> mul_ops) {
        this.factors = factors;
        this.mul_ops = mul_ops;
    }
}

File: jsParser/domain/Term.js
class Term {
  constructor (factors, mul_ops) {
    factors != null ? factors.length != 0 ? this.factors = factors : '' : ''
    mul_ops != null ? mul_ops.length != 0 ? this.mul_ops = mul_ops : '' : ''
  }
}
exports.Term = Term
